_install-and-configure/configuring-opensearch/index.md

```diff
@@ -25,13 +25,56 @@ Certain operations are static and require you to modify the `opensearch.yml` [co
 
 ## Specifying settings as environment variables
 
+You can specify environment variables in the following ways.
+
+### Arguments as start up
```

Comment:
Should "as" be "at"? Also, if "startup" is a noun, it should be one word with no space.